content/en/universal_service_monitoring/_index.md

```diff
@@ -821,6 +821,34 @@ DD_SYSTEM_PROBE_NETWORK_HTTP_REPLACE_RULES=[{"pattern":"<drop regex>","repl":""}
 
 {{< /tabs >}}
 
+### Non containerized support
```

Comment:
```suggestion
### Non-containerized support
```

Reply:
I'd suggest we change the title to: `Support for non-containerized services (Public Beta)`

Reply:
Lets also move this section to be immediately after **Exploring your services**

Reply:
@sunhay, the linter does not allow having Public Beta in the headline (capitalization issues)
is it enough to have the "public beta" message here?
![image](https://github.com/DataDog/documentation/assets/17148247/d49f53e6-2ce5-4b79-92b5-c99158dc4e60)

Reply:
I think that should be fine! @estherk15 any thoughts?

Reply:
Yes! This looks great!